cmd/create.go

```diff
@@ -22,16 +29,63 @@ func Create(projectName string, outDir string, t *templator.Templator) string {
 	rootDir := path.Join(outDir, projectName)
 	log.Println(aurora.Cyan(emoji.Sprintf(":tada: Creating project %s.", projectName)))
 	err := os.MkdirAll(rootDir, os.ModePerm)
-
```

Comment:
The command handlers should be kept slim and we should try to move all the logic deeper into some logical models. Love to plan out a clearer architecture for this project on Friday.

Reply:
Yeah I left a todo there